src/org/opendatakit/briefcase/ui/automation/AutomationPanel.java

```diff
@@ -117,6 +119,15 @@ private void generateScript(String scriptName, AutomationConfiguration configura
 
   }
 
+  private void writeBriefcaseJarTwoScript(List<String> scriptLines) {
```

Comment:
In this method, we're changing an input argument to have some sorts of output (we add elements to a list) but this is something we always want to avoid. 

You can avoid mutating the input argument by returning the line it produces, for example.

As a general rule, we always want our methods to behave like [referentially transparent functions](https://en.wikipedia.org/wiki/Referential_transparency?oldformat=true)